index.md

```diff
@@ -47,15 +47,19 @@ or maybe even a completely different approach.
    - The *Micro Architecture* considers decisions which may be taken
    individually for each module.
    
-4. The choice of *integration* options must be limited and standardized
+4. The choice of *integration and communication* options must be limited and standardized
    for the system. The integration might be done with synchronous or
    asynchronous communication, and/or on the UI level.
-   
-5. *Communication* must use a limited set of protocols like RESTful HTTP or
-   messaging. Also metadata, e.g. for authorization, must be
-   standardized. It might make sense to use just one protocol for each
+   Communication must use a limited set of protocols like RESTful HTTP or
+   messaging.
+   It might make sense to use just one protocol for each
    integration option.
    
+5. Metadata, e.g. for *authorization*, must be
```

Comment:
I would state first explain that - as users only want to login once / not for every module - authentication should be done centrally, while authorization should be done per module (as it requires domain knowledge). Then: as a consequence, metadata required for authorization must be standardized.

Reply:
Actually at the moment it talks about metadata for authorization but also e.g. for tracing. I would rather not put the point "no log in to each microservice" at the beginning because this point is broader.

Reply:
The point concerning authentication is discussed in https://isa-principles.org/reasoning.html . I would rather not discuss it here to keep the principles short.

Reply:
Ok, I see. I was probably wrongly referring this principle to the description/discussion under "Authentifizierung und Autorisierung" in https://www.innoq.com/de/articles/2018/09/prinzipien-fuer-unabhaengige-systeme/, probably as it is only mentioning "authorization" in bold letters. I see that this principle wants to be more general.

Reply:
Actually this is up for discussion. What the article says and what we discussed face-2-face is "authorization should be  a principle by itself". This is "metadata should be standardized. Authorization is an example" - because that is what the original principle actually said. We can still change it as discussed... 🙂